examples/render-md-server-side/README.md

```diff
@@ -0,0 +1,44 @@
+# Render Markdown from Anywhere Server-Side
+
+Remix supports MDX out of the box. This is great if you have your markdown/mdx files at compile time! If you are fetching your markdown files from a remote source, you have to work with unified or a wrapper such as react-remark.
+
+Unfortunately, react-remark does not support cjs anymore but Remix.run will not work with esm just yet (without async imports). The older versions of react-remark do not implement useRemarkSync. Thus, right now there is no way to use react-remark in remix to render markdown server-side since it will always requires async code (and useEffect).
+
+A solution: Make use of old versions of remark, unified, and rehype directly to basically reimplement react-remark for remix. Sounds complicated but it's really not!
+
+We wrap everything up in one small component that we then can use to render markdown across our application! We can even apply custom plugins and use custom React components for the output of our markdown! And the best: everything renders nicely on the server! 🚀
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/render-md-sync-dgdze)
```

Comment:
This CodeSandbox url should be something like https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/client-only-components using the repo and not a custom slug, that way clicking the link will open a sandbox with the lastest version of this example, with a custom CodeSandbox url you will have to keep the sandbox code updated with the example here

Reply:
Thanks for the explanation! I updated the url. It's not working now though, since the example is not present in the main branch. Just for review purposes, here is the link to the custom one: https://codesandbox.io/s/render-md-sync-dgdze